### DISCUSSION TITLE
Registered as a solo hacker but joined a project with a teammate

### DISCUSSION THREAD

**Discussion by YusC3** [microsoft/hack-together#70] [General]
Hello! I am a SDE participating in the hack. Excited to interact with you all :). I joined as solo but register with a teammate after finding one. How can I communicate that to adminisitrators?
Best,
Yusuf Corr

**Comment by waldekmastykarz** [comment]
Great to have you with us! Registrations are individual. When you submit your project using the Project issue template, add a list of everyone who's been working on it:

<img width="544" alt="image" src="https://user-images.githubusercontent.com/11164679/223693135-c52fc460-a354-465f-aaab-a2b02769e024.png">

Looking forward to seeing your hack!
